core/src/apps/cardano/helpers/bech32.py

```diff
@@ -6,6 +6,7 @@
 HRP_TESTNET_ADDRESS = "addr_test"
 HRP_REWARD_ADDRESS = "stake"
 HRP_TESTNET_REWARD_ADDRESS = "stake_test"
+HRP_PUBLIC_KEY = "ed25519_pk"
```

Comment:
this is actually not a CIP-0005 prefix: https://github.com/cardano-foundation/CIPs/blob/master/CIP-0005/CIP-0005.md but rather a jormungandr convention, so I'd name it HRP_JORMUN_PUBLIC_KEY and a source of it is e.g. this code I found: https://github.com/cardano-foundation/CIPs/blob/master/CIP-0005/CIP-0005.md

Reply:
Renamed in https://github.com/vacuumlabs/trezor-firmware/pull/57/commits/02a3b6f62c4f15bba6d5ed2f6ef0be7c19ee17e6. I also added comments with links to CIP-0005 and a [Jormungandr prefix source code](https://github.com/input-output-hk/voting-tools-lib/blob/18dae637e80db72444476606ab264b973bcf1a9d/src/Cardano/API/Extended.hs#L226) I found, since you've pasted the same link twice - is this link OK or which one have you found?

Reply:
here's the correct link for the jormungandr prefix source: https://github.com/input-output-hk/jormungandr/blob/a057af27493d823be02480bb20258c25ff979e2a/jormungandr-lib/src/crypto/key.rs#L126

Reply:
but the one you included is actually better